submissions/jeffersonMormah/javaScipt2/script.js

```diff
@@ -0,0 +1,18 @@
+var a = 1
+var b = 123;
+var c = 7;
+var d = b - a;
+
+for (var a = 1; a < d; a++){
```

Comment:
In this statement, your initialization should be less than or equal to the defined number.

Reply:
reinitialized to <=